iina/MPVTrack.swift

```diff
@@ -68,55 +68,47 @@ class MPVTrack: NSObject {
   var demuxFps: Double?
 
 
-  var readableTitle: String {
-    get {
-      return "\(self.idString) \(self.infoString)"
-    }
-  }
+  var readableTitle: String { "\(idString) \(infoString)" }
 
-  var idString: String {
-    get {
-      return "#\(self.id)"
-    }
-  }
+  var idString: String { "#\(id)" }
 
   var infoString: String {
     get {
       // title
-      let title = self.title ?? ""
+      let title = title ?? ""
       // lang
       let language: String
-      if let lang = self.lang, lang != "und", let rawLang = ISO639Helper.dictionary[lang] {
+      if let lang = lang, lang != "und", let rawLang = ISO639Helper.dictionary[lang] {
         language = "[\(rawLang)]"
       } else {
         language = ""
       }
       // info
       var components: [String] = []
-      if let ds = self.decoderDesc, let shortDs = ds.components(separatedBy: "(")[at: 0] {
-        components.append("\(shortDs.replacingOccurrences(of: " ", with: ""))")
+      if let codec = codec {
```

Comment:
Should we start using the [if let shorthand](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0345-if-let-shorthand.md) introduced in Swift 5.7?

Reply:
Agreed.